extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -565,12 +582,11 @@ angular.module('oppia').component('svgFilenameEditor', {
           color: 'rgba(0,0,0,1)'
         });
         ctrl.fabricjsOptions.size = '18px';
```

Comment:
Is this default? If so, use a descriptive variable name for '18px'.

Reply:
Explain why this needs to be set explicitly.

Reply:
Done.